src/pytorch_lightning/plugins/precision/deepspeed.py

```diff
@@ -97,24 +86,19 @@ def backward(
                 "You have overridden the `LightningModule.backward` hook but it will be ignored since DeepSpeed handles"
                 " the backward logic internally."
             )
+        optimizer, optimizer_idx, *args = args
         deepspeed_engine: "deepspeed.DeepSpeedEngine" = model.trainer.model
-        deepspeed_engine.backward(closure_loss, *args, **kwargs)
-
-    def _run_backward(
```

Comment:
Removed run_backward in all plugins. This is dead code now after #14735, and calls into the Lite specific backward implementation.